guice/pom.xml

```diff
@@ -50,4 +51,30 @@
 
     </dependencies>
 
+    <profiles>
+        <!-- Profile for non-JDK 8 -->
```

Comment:
what if we drop support for Java8 in this new version? (and I suggest we do, Java8 is soooo old)

Reply:
If we drop support for Java8, then we do not need to define this profile.